README.md

```diff
@@ -11,3 +11,83 @@ and call `bitrise run test`.
 
 *Check the `bitrise.yml` file for required inputs which have to be
 added to your `.bitrise.secrets.yml` file!*
+
```

Comment:
please use the new template

Reply:
Fixed in:
https://github.com/bitrise-steplib/steps-set-xcode-build-number/pull/15/commits/e0aea5408b0f5179ee8b6dcc4f757a3b7cfd1e69

Reply:
this is not the one :)

Reply:
Fixed